content/concepts/sampling_mode.md

```diff
@@ -5,11 +5,11 @@ disableToc: false
 chapter: false
 ---
 
-> Sampling mode makes it possible to apply the Core Rule Set to a limited percentage of traffic only. This may be useful in certain scenarios when enabling CRS for the first time, as this page explains.
+> Sampling mode makes it possible to apply the CRS to a limited percentage of traffic only. This may be useful in certain scenarios when enabling CRS for the first time, as this page explains.
```

Comment:
```suggestion
> Sampling mode makes it possible to apply CRS to a limited percentage of traffic only. This may be useful in certain scenarios when enabling CRS for the first time, as this page explains.
```